sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/src/TelemetryDebugWriter.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#if DEBUG
+
+using System.Diagnostics;
+
+namespace Microsoft.OpenTelemetry.Exporter.AzureMonitor
+{
+    internal class TelemetryDebugWriter
+    {
+        internal static void WriteTelemetry(string message)
+        {
+            if (message == null)
+            {
+                return;
+            }
+
+            if (Debugger.IsAttached && Debugger.IsLogging())
+            {
+                Debugger.Log(0, "category",  message);
```

Comment:
what is "category" ?

Reply:
It's just a string to categorize the message. It can be any value, I wanted to use `NULL` here. But in application insights we use "category" hence used the same here.

Reply:
Set it to `null`